src/Avatar/Avatar.js

```diff
@@ -77,6 +94,10 @@ Avatar.propTypes = {
    */
   alt: PropTypes.string,
   /**
+   * Used to render icon and text elements inside the Avatar.
+   */
+  children: PropTypes.node,
```

Comment:
What was the issue with the icon property? I mean, we could have simply renamed it children. Why keeping both or do we need a different logic?